packages/contracts/deploy/40_finalize-managing-dao/30_install-multisig-on-managing-dao.ts

```diff
@@ -15,9 +15,11 @@ const func: DeployFunction = async function (hre: EHRE) {
   // Get info from .env
   const approvers =
     process.env.MANAGINGDAO_MULTISIG_APPROVERS?.split(',') ||
-    (network.name !== 'mainnet' ? [deployer] : []);
+    (network.name === 'localhost' || network.name === 'hardhat'
+      ? [deployer]
+      : []);
   const minApprovals = parseInt(
-    process.env.MANAGINGDAO_MULTISIG_MINAPPROVALS || '1'
+    process.env.MANAGINGDAO_MULTISIG_MINAPPROVALS || '0'
```

Comment:
Is `minApprovals: 0` even a valid setting?

Reply:
not it is not valid to be 0, contract will revert